config/crd/patches/webhook_in_machinehealthchecks.yaml

```diff
@@ -0,0 +1,17 @@
+# The following patch enables conversion webhook for CRD
```

Comment:
we also need to add a block like https://github.com/kubernetes-sigs/cluster-api/blob/master/config/kustomization.yaml#L30-L35 to kustomization.yaml to patch the webhook namespace

Reply:
Have added 🙂